x-pack/plugins/infra/public/pages/metrics/hosts/hooks/use_hosts_table.tsx

```diff
@@ -221,6 +232,28 @@ export const useHostsTable = () => {
           },
         ],
       },
+      ...(displayAlerts
+        ? [
+            {
+              name: TABLE_COLUMN_LABEL.alertsCount,
+              field: 'alertsCount',
+              sortable: true,
+              'data-test-subj': 'hostsView-tableRow-title',
```

Comment:
nit: let's fix the width to something around `120px`, so that this column doesn't take up too much space, wdyt?

Reply:
Setting it to 120px is messing up the responsiveness of the table, I see that the only columns with absolute `width` value are rx and tx, is there a reason for just have it on this two?, and some of the big percentage values break a bit on smaller screens

https://github.com/elastic/kibana/assets/31922082/78f2f451-9fa4-4cbc-b36b-e1dd793835b2

Reply:
Yeah. I'm. not sure if rx and tx need to have fixed `width`. They look ok without it.

<img width="1199" alt="image" src="https://github.com/elastic/kibana/assets/2767137/e68d255e-193b-4411-9594-cfe8540dd2a0">

the responsiveness of the table is pretty bad already. Maybe we should look into that in a separate ticket?

<img width="747" alt="image" src="https://github.com/elastic/kibana/assets/2767137/e6bca177-6571-4b3c-8876-c630d7706724">

Reply:
Would be nice to have a `max-width` property for each column too, I think it'd be more useful than the `width`.

Reply:
> the responsiveness of the table is pretty bad already. Maybe we should look into that in a separate ticket?

Yes, let's not increase the scope of this issue